go.mod

```diff
@@ -108,6 +113,7 @@ require (
 	github.com/skratchdot/open-golang v0.0.0-20200116055534-eef842397966 // indirect
 	github.com/spf13/cobra v1.8.0 // indirect
 	github.com/spf13/pflag v1.0.5 // indirect
+	github.com/stretchr/testify v1.10.0
```

Comment:
@newera-begins - Sorry, just noticed it now. It would be better to maintain the testing practices same as https://github.com/openshift/managed-upgrade-operator/blob/master/docs/testing.md which is applicable for all the packages in the repo. Although it will work, we don't want to introduce separate testing practices via different tools at this point. Can you rework the test accordingly? Even a single test that's simple will suffice for now. So you can rebase your PR branch, rework the test and then push the changes.